components/email/src/Email.svelte

```diff
@@ -238,37 +238,28 @@
   // #region thread intake and set
   // The trick is to always ensure that activeThread is in the store; that way if we need to do fetches to update its messages, it too will be updated for free.
   // TODO: this feels like it could be a "$: activeThread =" reactive prop declaration instead of a conditional block. -Phil
-  $: if (thread && thread.id) {
```

Comment:
In the case where the email component fetched from the API using `MailboxStore.getThread(query)`, it would update the `$Threads` store, which would cause this binding to fire again (infinitely). This was only happening when an existing email component had already been loaded and another one was created in the same parent app.